packages/bot-web-ui/src/components/run-panel.jsx

```diff
@@ -98,6 +98,54 @@ const drawerFooter = ({
     );
 };
 
+const mobileDrawerFooter = ({
+    onStopButtonClick,
+    onRunButtonClick,
+    is_stop_button_visible,
+    is_stop_button_disabled,
+}) => {
+    return (
+        <div className='controls__section'>
+            <div className='controls__buttons'>
+                {is_stop_button_visible ? (
+                    <Button
+                        className='controls__stop-button'
+                        is_disabled={is_stop_button_disabled}
+                        text={localize('Stop bot')}
+                        icon={<Icon icon='IcPause' className='run-panel__button--icon' color='active' />}
+                        onClick={onStopButtonClick}
+                        has_effect
+                        primary
+                        large
+                    />
+                ) : (
+                    <Button
+                        className='controls__run-button'
+                        text={localize('Run bot')}
```

Comment:
As per the design, the text should only be `Run`.

```suggestion
                        text={localize('Run bot')}
```